Source/Core/WinUpdater/WinUI.cpp

```diff
@@ -182,7 +182,7 @@ void Error(const std::string& text)
   auto wide_text = UTF8ToWString(text);
 
   MessageBox(nullptr,
-             (L"A fatal error occured and the updater cannot continue:\n " + wide_text).c_str(),
+             (L"A fatal error occurred and the updater cannot continue:\n" + wide_text + "\nIf this persists, please redownload the latest update manually.").c_str(),
```

Comment:
You've now got a new build error (I guess the second string needs to be prefixed with `L` as well? I haven't worked with wide strings much), and there's still a lint error. I think the correct formatting is this:

```C++
  MessageBox(nullptr,
             (L"A fatal error occurred and the updater cannot continue:\n" + wide_text +
              L"\nIf this persists, please redownload the latest update manually.")
                 .c_str(),
             L"Error", MB_ICONERROR);
```